### PR TITLE
 feat(lang): add support for Traditional Chinese

### PR DESCRIPTION
**Pull request by yelandiealma**

### CODE REVIEW

File: Packages/enitimeago.non-destructive-mmd/Editor/Localization.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿using System.Linq;
```
Comment (enitimeago) [inline Packages/enitimeago.non-destructive-mmd/Editor/Localization.cs:1]: Sorry I think my lint GitHub Action was not working correctly, could you revert this change?

**Review by enitimeago — CHANGES_REQUESTED** [review]
Thank you very much for the contribution!

Before I merge can you undo the change in line 1 of Localization.cs? I think there is a BOM character that should not be removed.

**Comment by yelandiealma** [comment]
Is it okay?

**Review by enitimeago — APPROVED** [review]
LGTM, thank you!

**Comment by enitimeago** [comment]
This is now live as version 1.1.0-beta.3, thanks again!

**Comment by enitimeago** [comment]
@yelandiealma This is now live as version 1.1.0-beta.3, thanks again!

**Pull request merged**